test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -22,7 +26,7 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $null = $ps.AddScript({
             $obj = [PSCustomObject]@{P1 = ''; P2 = ''; P3 = ''; P4 = ''; P5 = ''; P6 = ''}
             $obj.P1 = $obj.P2 = $obj.P3 = $obj.P4 = $obj.P5 = $obj.P6 = $obj
-            1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10 -Verbose
+            1..100 | Foreach-Object { $obj } | ConvertTo-Json -Depth 10
```

Comment:
Surprised this test passed if you remove `-verbose` as line 35 is waiting on verbose output?

Reply:
The waiting take 10 seconds until timeout.

Reply:
You are removing `-Verbose` because another PR removed the verbose message (which you added back with the test hook in the other PR), right?  If so, I suppose this is ok, but it seems this change doesn't have any impact to this test so wondering why have this change in this PR at all?

Reply:
Reverted the change.